app/src/main/java/org/mifos/mobile/ui/fragments/HomeOldFragment.kt

```diff
@@ -271,6 +269,43 @@ class HomeOldFragment : BaseFragment(), HomeOldView, OnRefreshListener {
         }
     }
 
+    /**
+     * Toggles visibility of saving and loan total amounts on home,
+     * and updates the corresponding button icon.
+     */
```

Comment:
Remove this comment, please and extract all the toggle-related code to a sperate function and give that function an explanatory name.

Reply:
@jawidMuhammadi I've removed the comment and moved the toggle-related code to a separate function named toggleVisibilityButton.